packages/color-field/README.md

```diff
@@ -0,0 +1,108 @@
+## Description
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/color-field?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/color-field)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/color-field?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/color-field)
+
+```
+yarn add @spectrum-web-components/color-field
+```
+
+Import the side effectful registration of `<sp-color-field>` via:
+
+```
+import '@spectrum-web-components/color-field/sp-color-field.js';
+```
+
+When looking to leverage the `ColorField` base class as a type and/or for extension purposes, do so via:
+
+```
+import { ColorField } from '@spectrum-web-components/color-field';
+```
+
+## Example
+
+### Default
+
+```html
+<sp-color-field></sp-color-field>
+```
+
+### devMode
```

Comment:
This doesn't seem current to the available API?

Reply:
Done!